Core/Credentials.cs

```diff
@@ -1,16 +1,16 @@
 ﻿namespace Core
 {
-    public class Credentials
-    {
-        public Credentials(string cardNumber, string merchantPassword, string merchantId)
-        {
-            CardNumber = cardNumber;
-            MerchantPassword = merchantPassword;
-            MerchantId = merchantId;
-        }
+	public class Credentials
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)